ni-oe-init-build-env

```diff
@@ -133,6 +133,9 @@ export PYREX_CONFIG_BIND="${NILRT_ROOT}"
 export PYREX_OEINIT="${NILRT_ROOT}/sources/openembedded-core/oe-init-build-env"
 export PYREX_ROOT="${NILRT_ROOT}/sources/pyrex"
 
+export PYREX_BUILD_NILRT_IMAGE="${PYREX_BUILD_NILRT_IMAGE:-build-nilrt:latest}"
```

Comment:
Once a release branch is created, should the default be updated to the respective release-tagged container? Like what you are doing in the sumo changelist.

Reply:
Or why not use the same logic used for the sumo branch?